pkg/kubelet/cm/cpumanager/policy_none.go

```diff
@@ -17,9 +17,10 @@ limitations under the License.
 package cpumanager
 
 import (
+	"context"
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I guess this is done by your IDE. Not related to the PR scope.

Reply:
fixed, thanks